contrib/julia-config.jl

```diff
@@ -51,6 +51,8 @@ end
 function ldlibs()
     if is_unix()
         return replace("""-Wl,-rpath,$(libDir()) -ljulia""","\\","\\\\")
+    elseif is_windows()
+        return replace("""-ljulia -lopenlibm""","\\","\\\\")
```

Comment:
looks like the replacement doesn't do anything? also why have this as a new case?